fe1-web/src/core/components/QRCode.tsx

```diff
@@ -21,6 +21,7 @@ const styles = StyleSheet.create({
       and a height of 50%. centering this gives us 25% margin on every side
     */
     position: 'absolute',
+    padding: '1%',
```

Comment:
Is there a reason you use percentage padding? The overlay is made in percentages to ensure decoding will still work but I think you should be able to use non-relative padding here (i.e. Spacing.x1) since the font size is also not relative to the size of the qr code.

Reply:
Using `Spacing.x` is less elegant for me, since the padding is related to the size of the blue circle, which is directly related to the hard-coded size of the QRCode in the QRCode.txs file. In addition, the text styles used inside the QRCode are not defined using `Spacing.x`.

Using a constant padding instead of a relative is indeed better, and this change will be implemented in the next commit. 

Btw, using the `centeredBase` style instead of `base` and `centered` gives a too large padding for potential larger texts later.

Reply:
Hmm I see your point 🤔

But you should not add the px suffix, react native uses unitless values :) (a bit weird, yes)
https://reactnative.dev/docs/height-and-width

And you‘re right, somehow (probably it was even me) there is an additional margin for centeredBase 😅